frontend/src/pages/Landing/index.css

```diff
@@ -250,14 +250,10 @@ a.desc {
   background: #c5c5c5;
 }
 
-#caughtUp {
```

Comment:
You could remove the ```.smart-hub--caught-up``` class entirely and replace it with helper classes. I don't think you can match padding exactly but those values look fairly arbitrary. Also don't think we need ```white-space: nowrap;```. I'd try removing that class and this css and replacing it with like ```text-center padding-10``` or something like that.

Reply:
I have updated the CSS and used the helper class.